apollo-router/src/axum_http_server_factory.rs

```diff
@@ -511,10 +523,88 @@ fn prefers_html(accept_header: &HeaderValue) -> bool {
 }
 
 #[derive(Clone)]
-struct PropagatingMakeSpan(DefaultMakeSpan);
+struct PropagatingMakeSpan {
+    level: Level,
+}
+
+impl PropagatingMakeSpan {
```

Comment:
Suggest not parameterizing this. We will only ever want Info.
This removes the need for the macro stuff later.